cypress/support/server.ts

```diff
@@ -595,11 +596,7 @@ export const mockPostItemMembership = (
   cy.intercept(
     {
       method: HttpMethod.Post,
-      url: new RegExp(
-        `${API_HOST}/${parseStringToRegExp(
-          buildPostItemMembershipRoute(ID_FORMAT),
-        )}`,
-      ),
+      url: `${API_HOST}/item-memberships?*`,
```

Comment:
I think you need to escape the `?` no ? 
```suggestion
      url: `${API_HOST}/item-memberships\?*`,
```

Reply:
No I don't think it is necessary:

`Unnecessary escape character: \?.eslint`

Reply:
Okay good then !